example/README.md

```diff
@@ -0,0 +1,51 @@
+Example Webpack setup
+======================
+
+Here you'll find an example Webpack configuration. The `webpack.config.js` here will output two bundles here: `application.js` and `vendor.js`, and will process an image.
+
+## Starting
+
+Clone the plugin and run `npm install` in the root folder and in the `example/` folder. Then start `npm run` to start Webpack. In the folder `build/` you should see four files:
+
+1. `application.js`,
+2. `vendor.js`,
+3. `963eb32907744d9a0d6b98127162808f.svg` (fingerprint will be different),
+4. and `manifest.json` file.
+
+## Basic configuration
+
+The configuration is consited out of 4 main parts:
+
+### 1. `entry`
+
+Here you define where are the JavaScript files where you will require all other JS code, SCSS (not in this example), and other assets. They are named here `application` and `vendor`. That are the names that they will take when outputed.
+
+### 2. `output`
+
+This is where you define the output path for all the Webpack generated files. Here your JS, SCSS, and other asset files will end up after the Webpack finishes.
```

Comment:
Webpack-generated

"Webpack will store the resulting JS, SCSS, and other asset files in the given directory after processing them" (or something along that vein)